packages/venia-concept/src/components/Main/main.css

```diff
@@ -5,18 +5,8 @@
     z-index: 1;
 }
 
-.page {
-    min-height: 100vh;
-}
```

Comment:
Why would we remove this?

Reply:
Adding it back since we're overlaying the notification. 
If you were curious though:
![100vh](https://user-images.githubusercontent.com/6378569/49180420-6df79f00-f309-11e8-964e-a9aca43fc114.gif)